moonsheep/tests/test_settings.py

```diff
@@ -8,12 +10,25 @@
     'django.contrib.contenttypes',
 ]
 
+
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': ':memory:',
     }
 }
 
+if 'TRAVIS' in os.environ:
```

Comment:
Lookgs good. I guess that at some point we should inject database connection url via env.